src/clarity-angular/dropdown/_dropdown.clarity.scss

```diff
@@ -177,6 +174,20 @@ $dropdown-white: clr-getColor(lightest);
             }
         }
 
+        .dropdown-item {
```

Comment:
I think we also need to add the `.btn` selector (but only inside `.dropdown-menu`) here. Its because in button groups sometimes `.btn` appears inside of the `.dropdown-menu` and it doesn't have the `.dropdown-item` class on it.

For eg:
![image](https://cloud.githubusercontent.com/assets/1426805/24910823/d59fabe6-1e7d-11e7-9745-2e3317f261be.png)